src/sentry/tasks/post_process.py

```diff
@@ -278,23 +276,27 @@ def post_process_group(
                     safe_execute(callback, event, futures, _with_transaction=False)
 
             try:
-                lock = locks.get("w-o:{}-d-l".format(event.group_id), duration=10,)
-                try:
-                    with lock.acquire():
-                        has_commit_key = "w-o:{}-h-c".format(event.project.organization_id)
-                        org_has_commit = cache.get(has_commit_key)
-                        if org_has_commit is None:
-                            org_has_commit = Commit.objects.filter(
-                                organization_id=event.project.organization_id
-                            ).exists()
-                            cache.set(has_commit_key, org_has_commit, 3600)
-
-                        if org_has_commit and features.has(
-                            "organizations:workflow-owners", event.project.organization,
-                        ):
-                            process_suspect_commits(event=event)
-                except UnableToAcquireLock:
-                    pass
+                has_commit_key = "w-o:{}-h-c".format(event.project.organization_id)
+                org_has_commit = cache.get(has_commit_key)
+                if org_has_commit is None:
+                    org_has_commit = Commit.objects.filter(
+                        organization_id=event.project.organization_id
+                    ).exists()
+                    cache.set(has_commit_key, org_has_commit, 3600)
+
+                if org_has_commit and features.has(
+                    "organizations:workflow-owners", event.project.organization,
+                ):
```

Comment:
2/2: We should move the debounce logic to below this `if` statement and before the call to `get_frame_paths` to avoid the extra cycles of attr extraction.